docs.json

```diff
@@ -79,6 +76,17 @@
     ]
   },
   "footer": {
+    "links": [
+      {
+        "title": "Resources",
+        "items": [
+          {
+            "label": "Gnosis Pay Media Kit",
+            "href": "https://github.com/gnosispay/docs/tree/main/static/Gnosis%20Pay%20Media%20Kit"
```

Comment:
Do we need the media kit hosted on this repo? As an alternative we can use a Google drive link or something and not get this large files on our git.